mailet/base/src/main/java/org/apache/mailet/base/MailetUtil.java

```diff
@@ -109,4 +122,90 @@ public static boolean getInitParameter(MailetConfig config, String name, boolean
         }
         return result;
     }
+
+    public static boolean canSendAutomaticResponse(Mail mail) throws MessagingException {
```

Comment:
ok about reuse. Naming is still wrong : I mean, it's more "should" than "can".
About the class location, It's bad too. What is the goal of MailUtil ? I guess there are several